curriculum/challenges/english/25-front-end-development/lecture-understanding-form-validation/6733d3ab69e94b7df7ee91b0.md

```diff
@@ -10,6 +10,81 @@ dashedName: how-does-the-submit-event-work-with-forms
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+How does the submit event work with forms?
+
+Let’s learn about how the submit event works with HTML forms. First, we need to understand *how* to submit a form. There are three ways a form can be submitted.
+
+The first is when the user clicks a button in the form which has the `type` attribute set to `submit`.
+
+The second is when the user presses the `Enter` key on any editable input field in the form.
```

Comment:
```suggestion
The second is when the user presses the Enter key on any editable input field in the form.
```
I think this shouldn't be surrounded by backticks since it's a key with different names depending on the language.